docs/terms.md

```diff
@@ -0,0 +1,26 @@
+# What are the terms we use and what do they mean?
+
+### Group
+A group is a collection of unit files tied together. This can be seen as
+equivalent to kubernetes pods. Units within a Inago group are scheduled
```

Comment:
yes, that is the assumption we do on what the user provides us. Might just be a small phrasing issue, as this felt on first read like sthg that is enforced by Inago.